pkg/controller/cluster/resource/resource.go

```diff
@@ -350,10 +350,34 @@ func StatefulSetForRack(r scyllav1alpha1.RackSpec, c *scyllav1alpha1.Cluster, si
 		},
 	}
 
+	sysctlContainer := sysctlInitContainer(c.Spec.Sysctls)
+	if sysctlContainer != nil {
+		sts.Spec.Template.Spec.InitContainers = append(sts.Spec.Template.Spec.InitContainers, *sysctlContainer)
+	}
 	sts.Spec.Template.Spec.Containers = append(sts.Spec.Template.Spec.Containers, *agentContainer(c))
 	return sts
 }
 
+func sysctlInitContainer(sysctls []string) *corev1.Container {
```

Comment:
It would be better to call it sysctlSetContainer or vsysctlContainer to avoid confusion with k8s init containers.

Reply:
Sure.

Reply:
No I mean why? These are k8s init containers.

Reply:
Correct.